devel/ninja/Portfile

```diff
@@ -32,9 +37,20 @@ homepage            https://ninja-build.org
 checksums           rmd160  6765fd1becaccc480b857bdf7953405dfdd43ff3 \
                     sha256  f0055ad0369bf2e372955ba55128d000cfcc21777057806015b45e4accbebf23 \
                     size    292098
+platform darwin 8 {
+    github.setup        ninja-build ninja 1.12.1 v
+    revision            1
+    checksums           ninja-1.12.1.tar.gz \
+                        rmd160  edb99cb67641947a6cdde89300a326de69df77f4 \
+                        sha256  821bdff48a3f683bc4bb3b6f0b5fe7b2d647cf65d52aeb63328c91a6c6df285a \
+}
+
 github.tarball_from archive
 checksums-prepend   ${distfiles}
 set gtest_version   1.16.0
+platform darwin 8 {
```

Comment:
It will be easier to maintain (both for 10.4 and for 10.5+) if all 10.4-specific code goes into a single block somewhere below. If you prefer, I can handle that, you can just test the build then.